build.gradle.kts

```diff
@@ -95,6 +95,15 @@ val testIntegration = task<Test>("testIntegration") {
     maxParallelForks = 1
 }
 
+val testE2E = task<Test>("testE2E") {
```

Comment:
As this will be now a bamboo job, have you thought about reports generated? Or only logs?